### PR TITLE
feat(LightRegularGrid): Added IO ofr LightRegularGrid in Geode format.

### PR DESCRIPTION
**Pull request by MelchiorSchuh**

### CODE REVIEW

File: include/geode/mesh/io/light_regular_grid_input.h [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+namespace geode
+{
+    /*!
+     * API function for loading an LightRegularGrid.
+     * The adequate loader is called depending on the filename extension.
+     * @param[in] filename Path to the file to load.
+     */
+    template < index_t dimension >
+    LightRegularGrid< dimension > load_light_regular_grid(
```
Comment (BotellaA) [inline include/geode/mesh/io/light_regular_grid_input.h:52]: Do we use 2D-3D alias also for this api?

File: include/geode/mesh/io/light_regular_grid_input.h [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        using Base::InputData;
+        using Base::MissingFiles;
+
+        LightRegularGridInput( absl::string_view filename ) : Base{ filename }
+        {
+        }
+
+        LightRegularGrid< dimension > read() final
+        {
+            std::ifstream file{ to_string( this->filename() ),
```
Comment (BotellaA) [inline include/geode/mesh/io/light_regular_grid_input.h:69]: to move in cpp

File: include/geode/mesh/io/light_regular_grid_output.h [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    public:
+        LightRegularGridOutput( absl::string_view filename )
+            : Output< LightRegularGrid< dimension > >{ filename }
+        {
+        }
+
+        std::vector< std::string > write(
+            const LightRegularGrid< dimension >& grid ) const final
+        {
+            std::ofstream file{ to_string( this->filename() ),
```
Comment (BotellaA) [inline include/geode/mesh/io/light_regular_grid_output.h:69]: to move in cpp

**Review by BotellaA — APPROVED** [review]

**Comment by panquez** [comment]
Be aware of typo the commit name

**Pull request merged**